decide/test_rust.py

```diff
@@ -0,0 +1,7 @@
+import rust
```

Comment:
where does it get this?  there must be something in the requirements.txt, no?

Reply:
i think `maturin develop` does the magic here. (https://github.com/PyO3/maturin)

Reply:
it is defined in Cargo.toml: 
```
[lib]
name = "rust"
```